literature/literature.bib

```diff
@@ -25245,17 +25335,6 @@ @inproceedings{LH:PLDI04
 	numpages  = {12},
 }
 
-@inproceedings{TBW:PPPJ04,
```

Comment:
I am missing a comment at the duplicate entry like this:

Formerly known as TBW:PPPJ04

Reply:
as i am the only person using this entry, and it's only a few month old, I'd omit it in this case